Makefile

```diff
@@ -407,3 +407,44 @@ consistent-crds-manifests-upstream:
 	$(Q)cd ./manifests-upstream/${OPERATOR_VERSION}/ && ln -srf ../../deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml \
 	servicebindingrequests.apps.openshift.io.crd.yaml
 
+## -- Target for merge to master dev release --
+.PHONY: merge-to-master-release
+## Make a dev release on every merge to master
+merge-to-master-release:
+	echo "${QUAY_TOKEN}" | docker login -u "redhat-developer+travis" --password-stdin quay.io
+	$(eval COMMIT_COUNT := $(shell git rev-list --count HEAD))
+	$(Q)operator-sdk build \
+	"$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)"
+	#docker login -u="redhat-developer+travis" -p=${QUAY_TOKEN}
+	docker tag "$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)" \
+	"$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)"
+	docker push "$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)"
+
+
+## -- Target for pushing manifest bundle to service-binding-operator-manifest repo --
+.PHONY: push-to-manifest-repo
+## Push manifest bundle to service-binding-operator-manifest repo
+push-to-manifest-repo:
+	@rm -rf $(MANIFESTS_TMP) || true
+	@mkdir -p ${MANIFESTS_TMP}/${BUNDLE_VERSION}
+	operator-sdk olm-catalog gen-csv --csv-version $(BUNDLE_VERSION)
+	go build -ldflags "-X main.BundleVersion=$(BUNDLE_VERSION)" ./hack/add-info-to-csv.go
+	./add-info-to-csv
```

Comment:
Let's ensure everything that we have in the latest CSV here https://github.com/operator-framework/community-operators/tree/master/community-operators/service-binding-operator currently is copied over, as used a starting point.

Reply:
Since the PR is to the master branch, and the master is at v0.0.20, so the version has been kept as 0.0.20 (taken from the community -operators)

Reply:
What would be the generated bundle version on merge to master?

Reply:
0.0.20-{COMMIT_COUNT}